model/lists/ListService.php

```diff
@@ -70,10 +71,14 @@ public function getListElement(RdfClass $listClass, $uri)
             : iterator_to_array($result->getIterator())[0];
     }
 
-    public function getListElements(RdfClass $listClass, $sort = true, $limit = 0)
+    public function getListElements(RdfClass $listClass, $sort = true, $limit = 0, $dataLang = null)
```

Comment:
Although it is allowed by php, I do not recommend change the signature of method declared in the parent class, cause it breaks `Liskov substitution` principle. I would rather change the parent class too or better, create a new method to avoid more inconsistencies.